UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>Can you add a markdown cell above this code block that explains what you are doing? I can follow along that you are specifying the coordinates you are looking for and you want results in a radius of 10 arcseconds. But, for example, will the user always use that URL to search GALEX though VO? Can you link to documentation to where a new user would have found that URL in the first place?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>